$NetBSD: patch-configure,v 1.3 2017/07/12 13:56:00 hauke Exp $

Disable zeroconf explicitely, unless enabled - the distribution
appears not to allow for this.

We do not at this point support zeroconf via avahi. Since configure
does not allow for building without avahi, and native avahi is picked
up on some Linuxen, we disable all avahi tests unconditionally here.

Fix build on SunOS by disabling the isainfo bitness check 

--- configure.orig	2013-07-23 09:11:35.000000000 +0000
+++ configure
@@ -16109,7 +16109,7 @@ fi
 if test "${enable_zeroconf+set}" = set; then :
   enableval=$enable_zeroconf; zeroconf=$enableval
 else
-  zeroconf=try
+  zeroconf=no
 
 fi
 
@@ -16185,7 +16185,8 @@ $as_echo "#define HAVE_MDNS 1" >>confdef
         fi
 
         # mDNS support using Avahi
-        if test x"$found_zeroconf" != x"yes" ; then
+        #if test x"$found_zeroconf" != x"yes" ; then
+	if 0 ; then
             ac_fn_c_check_header_mongrel "$LINENO" "avahi-client/client.h" "ac_cv_header_avahi_client_client_h" "$ac_includes_default"
 if test "x$ac_cv_header_avahi_client_client_h" = xyes; then :
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for avahi_client_new in -lavahi-client" >&5
@@ -17535,7 +17536,7 @@ $as_echo_n "checking if we have to build
 	   # check for isainfo, if not found it has to be a 32 bit kernel (<=2.6)
 	   if test -x /usr/bin/isainfo; then
 		# check for 64 bit platform
-		if isainfo -kv | grep '^64-bit'; then
+		if test "$MACHINE_ARCH" = "x86_64"; then
 			COMPILE_64BIT_KMODULE=yes
 		fi
 	   fi
